app/__main__.py

```diff
@@ -53,7 +100,11 @@ async def message(interaction: discord.Interaction, message: str) -> None:
 @client.tree.command()
 async def upgrade(interaction: discord.Interaction) -> None:
     """Upgrade."""
-    await interaction.response.send_message("Upgraded")
+    embed = discord.Embed(title="Upgrade menu", description="Select an upgrade to obtain")
+    embed.add_field(name="Better CPS", value="Increase the amount of characters you can send per second")
+    embed.add_field(name="Higher Priority", value="Increase the priority of your messages")
+    view = UpgradeView()
+    await interaction.response.send_message(embed=embed, view=view)
```

Comment:
TBH should the upgrade menu say costs (just put "[cost]" for now) and be ephemeral? That might be nicer, what do you think?

Reply:
If so, then my first comment about sending a new message instead of editing might be incorrect: I think in that case you should just edit the embed.